cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-tool-with-recurring-tool-properties.txt

```diff
@@ -1,4 +1,14 @@
-ERROR [Ballerina.toml:(6:1,10:24)] recurring target module 'delivery' found in Ballerina.toml. Target module must be unique for each tool
-ERROR [Ballerina.toml:(6:1,10:24)] recurring tool id 'generate-delivery-client' found in Ballerina.toml. Tool id must be unique for each tool
-WARNING [Ballerina.toml:(12:1,15:24)] missing key '[targetModule]' in table '[tool.openapi]'. Default module will be taken as target module.
-WARNING [Ballerina.toml:(17:1,20:26)] tool options validation skipped due to: Schema file not found: grpc-options-schema.json
+ERROR [Ballerina.toml:(12:1,15:24)] recurring tool id 'generate-delivery-client' found in Ballerina.toml. Tool id must be unique for each tool
+ERROR [Ballerina.toml:(17:1,20:26)] recurring target module found in Ballerina.toml. Target module must be unique for each tool
+
+Executing Build Tools
+	openapi(generate-delivery-client)
+
+ERROR [openAPI sample build tool:(1:1,1:1)] The provided filePath does not exist
+WARNING: Skipping execution of build tool openapi(generate-delivery-client) as Ballerina.toml contains errors
+
```

Comment:
Do we need new line between warnings?

Reply:
This again was added to have a separation between the tool execution and compilation logs